library-best-practices/JLBP-3.md

```diff
@@ -0,0 +1,45 @@
+[JLBP-3] Use Semantic versioning
+--------------------------------
+
+Definition of semver: https://semver.org
+
+- For the purpose of compliance with semver, the stable surface of a library
+  serves as its "Public API." Annotations can be used to mark features
+  (classes, methods, etc) as unstable, and thus not part of the Public API,
+  such that semver rules don't have to apply. 
+  - Examples of annotations:
+    - Guava uses `@Beta`
```

Comment:
Could we standardize/recommend only one of these? We shouldn't have three different ways of marking something beta.

Reply:
That seems like a good idea, but not within the scope of this best practice. If we do it, we can come back and update this recommendation.